frontend/src/app/pages/Leaderboards/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { Leaderboards } from '../../components/Leaderboards/Leaderboards'
+import { Subtitle, Box } from './../../../styles/blocks'
+
+export const LeaderboardsPage = () =>
+  <>
```

Comment:
this doesn't need to be enclosed in a fragment bytw ;)

Reply:
[Fixed](https://github.com/salsita/foosball-rating/pull/92/commits/d7c17da31ec421acbb90f4a19ce22c34412cdf35#diff-c5555fa63b1f2efff1b0b492aff5c461)